lte/gateway/fabfile.py

```diff
@@ -219,7 +219,8 @@ def s1ap_setup_cloud():
 
 def integ_test(
     gateway_host=None, test_host=None, trf_host=None,
-    destroy_vm='True', provision_vm='True',
+    destroy_vm='True', build=None, hash=None,
```

Comment:
maybe instead of `build` you can use something like `useBuildFromRepo` or `use_repo` or something different that `build` 

The reason for that is for cwf integ test we already have build flag. That flag means that the test is run with whatever is installed on the vm. That is used for testing when you are building specific services on agw locally . That allow us to run the test without building the whole thing.   Maybe in the future we should add that `build` behaviour to  `lte_integ_test` too. So it would be good to be consistent

Reply:
In fact, maybe you can combine both flags. IF `magmacore_hash` is not null, that means you want to build from repo (?)

Reply:
yes that's the plan my dear Oriol. It will be changed when I implement the install_magma_from_registry function, right now it's just calling the _install_from_registry that dynamically grab the hash and install the package 

In the future I want dev to be able to do `fab install_magma:hash=<MY_HASH>` and have their built version tested locally